Rubberduck.Parsing/Symbols/ParsingExceptions/MainParseExceptionErrorListener.cs

```diff
@@ -12,7 +12,7 @@ public MainParseExceptionErrorListener(string componentName, ParsePass parsePass
         public override void SyntaxError(IRecognizer recognizer, IToken offendingSymbol, int line, int charPositionInLine, string msg, RecognitionException e)
         {
             // adding 1 to line, because line is 0-based, but it's 1-based in the VBE
-            throw new MainParseSyntaxErrorException(msg, e, offendingSymbol, line, charPositionInLine + 1, ComponentName, ParsePass);
+            //throw new MainParseSyntaxErrorException(msg, e, offendingSymbol, line, charPositionInLine + 1, ComponentName, ParsePass);
```

Comment:
This one was throwing all the time on my 4.7 update. Could you give me an overview of why is it a good idea to skip it? Just need some context.